### ISSUE TITLE
Scalactic's Every (de)serialization failing

### ISSUE THREAD

**Issue by pjan** [scalatest/scalatest#663]
I'm trying to use `Scalactic` as an opinionated either for my services in a clustered akka environment. When sending a `A Or Every[B]` over the wire however, I get the following exception:

```
java.io.InvalidClassException: org.scalactic.One; no valid constructor
```

The solution for now is to convert the `Every` to a list right before transmission, which breaks the flow. Is there a better way to handle this? Or can this be fixed?

**Comment by bvenners** [comment]
I never thought to make them Serializable. That's probably all that is needed. I unfortunately may not be able to do that for another 1.5 weeks, but as soon as I can I'll make that change and make another milestone release.

**Comment by pjan** [comment]
Thanks Bill. Much appreciated!

**Comment by cheeseng** [comment]
FYI, I submitted the following PRs: 

for 2.2.x: 
https://github.com/scalatest/scalatest/pull/684

for 3.0.x: 
https://github.com/scalatest/scalatest/pull/685

Hopefully they will be included in upcoming 2.2.x and 3.0.x release soon.

**Comment by bvenners** [comment]
This was fixed five months ago and since then has been released both in 2.2.6 and 3.0.0-M15, but we never closed the issue. Closing now.
